etc/solr-config/access/conf/schema.xml

```diff
@@ -216,6 +216,8 @@
     <field name="rollup" type="string" indexed="true" stored="true" required="true" docValues="true"/>
     <!-- Stores publication status and other statuses that affect viewing of this object outside of group ACLs -->
     <field name="status" type="string" indexed="true" stored="true" multiValued="true"/>
+    <!-- Stores streaming URL for files with streaming property -->
+    <field name="streamingUrl" type="string" indexed="true" stored="true"/>
```

Comment:
We'll need to also update boxc-ansible with this

Reply:
thanks for the reminder! https://gitlab.lib.unc.edu/cdr/boxc-ansible/-/merge_requests/271